libunwind/src/UnwindRegistersRestore.S

```diff
@@ -703,6 +703,20 @@ Lnovec:
 
 #elif defined(__aarch64__)
 
+#if defined(__CHERI_PURE_CAPABILITY__)
+DEFINE_LIBUNWIND_FUNCTION(__rtld_unw_setcontext)
+  mov    c16, c3
+  ldp    c3, c4,  [c4, #0x030]
```

Comment:
The offset should be negative given that there was
```
add c4, c0, #0x210
```
before?

Reply:
I think so, yes.